### PR TITLE
Fix launching LS with IntelliJ 2022.3

### PR DESCRIPTION
**Pull request by evie-lau**
Fix another warning/exception with ActionPlaces

Tested and runs properly on 2022.3 in MacOS and Windows

### CODE REVIEW

**Review by kathrynkodama — APPROVED** [review]
LGTM, thanks @evie-lau

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <depends>com.intellij.properties</depends>
     <depends>org.jetbrains.idea.maven</depends>
     <depends>com.intellij.gradle</depends>
-    <idea-version since-build="222" until-build="222.*"/>
+    <idea-version since-build="222"/>
```
Comment (kathrynkodama) [inline src/main/resources/META-INF/plugin.xml:32]: Should we preemptively block for anything past 223.* or wait to see if it breaks us?

Comment (evie-lau) [reply src/main/resources/META-INF/plugin.xml:32]: I was considering that as well. Given how unstable LSP integration with IntelliJ is, might be better to add it in and bump when available and working.

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
     <depends>com.intellij.properties</depends>
     <depends>org.jetbrains.idea.maven</depends>
     <depends>com.intellij.gradle</depends>
-    <idea-version since-build="222" until-build="222.*"/>
+    <idea-version since-build="222" until-build="223"/>
```
Comment (kathrynkodama) [inline src/main/resources/META-INF/plugin.xml:32]: Do we have to use `223.*` to include all versions of 223? Looking at https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html seems like yes?

Comment (evie-lau) [reply src/main/resources/META-INF/plugin.xml:32]: 👍🏼 Seems like a safe bet.

**Review by kathrynkodama — APPROVED** [review]

**Pull request merged**